test/suites/storage_local_volume_handling.sh

```diff
@@ -66,6 +66,18 @@ test_storage_local_volume_handling() {
     lxc storage volume set "${pool}" vol1 user.foo=snap0
     lxc storage volume set "${pool}" vol1 snapshots.expiry=1H
 
+    lxc storage volume create "${pool}" blockVol --type=block
+    truncate -s 25MiB foo.iso
+    lxc storage volume import "${pool}" ./foo.iso isoVol
+
+    # security.shared is only allowed for block volumes
+    ! lxc storage volume set "${pool}" vol1 security.shared true || false
+    ! lxc storage volume set "${pool}" isoVol security.shared true || false
```

Comment:
So does `security.shared` not work for iso volumes because they are allowed to be attached concurrently to multiple instances (as this should be allowed because they are readonly)?

Reply:
Yes, they are sharable by default